Soomla/Assets/Plugins/Soomla/Profile/Domain/UserProfile.cs

```diff
@@ -80,48 +80,48 @@ protected UserProfile(Provider provider, string profileId, string username)
 			this.Username = JSONObject.DecodeJsString(jsonUP[PJSONConsts.UP_USERNAME].str);
 			this.ProfileId = JSONObject.DecodeJsString(jsonUP[PJSONConsts.UP_PROFILEID].str);
 
-			if (jsonUP[PJSONConsts.UP_FIRSTNAME]) {
```

Comment:
@EugeneButusov, I got this point, but I mean another... What if `jsonUP[PJSONConsts.UP_FIRSTNAME]` is `null`? Is it possible? 
It's not ObjC world :)